packages/wonder-blocks-core/src/hooks/__tests__/use-force-update.test.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import * as React from "react";
+import {render, act} from "@testing-library/react";
+import {renderHook} from "@testing-library/react-hooks";
+
+import {useForceUpdate} from "../use-force-update.js";
+
+describe("#useForceUpdate", () => {
+    it("should return a function", () => {
+        // Arrange
+
+        // Act
+        const {
+            result: {current: result},
+        } = renderHook(() => useForceUpdate());
+
+        // Assert
+        expect(result).toBeInstanceOf(Function);
+    });
+
+    describe("returned function", () => {
+        beforeEach(() => {
+            jest.useFakeTimers();
+        });
+
+        it("should cause component to render", () => {
```

Comment:
It might be good to have a test that checks that calling `forceUpdate()` multiple times will result in multiple re-renders.

Reply:
This test does just that.